src/palette/internal/palettecell.cpp

```diff
@@ -99,6 +99,7 @@ const char* PaletteCell::translationContext() const
     case ElementType::BREATH:
     case ElementType::FERMATA:
     case ElementType::SYMBOL:
+    case ElementType::HARP_DIAGRAM:
```

Comment:
For `ElementType::HARP_DIAGRAM`, this method should return "palette" (it should correspond with:
```c++
sp->appendElement(pedalDiagram, QT_TRANSLATE_NOOP("palette", "Harp pedal diagram"));
                                                   ^~~~~~~
```
_and_
```c++
sp->appendElement(pedalTextDiagram, QT_TRANSLATE_NOOP("palette", "Harp pedal text"));
                                                       ^~~~~~~
```